### PR TITLE
Fix test for `@LocalArmeriaPort(s)`

### PR DESCRIPTION
**Pull request by heowc**
Motivation:

In Spring Framework, `@LocalServerPort` values are not injected into the object to be made into a bean. However, `@LocalArmeriaPort` and `@LocalArmeriaPorts` in Armeria may have to be changed because such tests exist.

Modification:
- Fixed LocalArmeriaPortTest
- Fixed LocalArmeriaPortsTest
- Fixed typo

Result:
Fixed the wrong test.

### CODE REVIEW

**Review by trustin — APPROVED** [review]
Thanks, @heowc 😄

**Review by ikhoon — APPROVED** [review]
Thanks! @heowc

File: spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaBeanPostProcessor.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ private InjectionMetadata buildLocalArmeriaPortMetadata(Class<?> clazz) {
         return new InjectionMetadata(clazz, elements);
     }
 
-    private void serServer(Server server) {
+    private void setServer(Server server) {
```
Comment (minwoox) [inline spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaBeanPostProcessor.java:147]: 😱

**Review by minwoox — APPROVED** [review]
Thanks!

File: spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaBeanPostProcessor.java (hunk 1 above)
Comment (heowc) [reply spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaBeanPostProcessor.java:147]: Hahaha...😱

**Pull request merged**